File: Indixpert-Sept-batch2025/C Languege/array2/task5.c
#include <stdio.h>
int main()
{
    int array[5], i;
    int count = 0;

    for (i = 0; i < 5; i++)
    {
        printf("Enter Number = ");
        scanf("%d", &array[i]);
        count++;
        printf("%d", count);
    }
    printf("Total Element %d", count);
}

File: Indixpert-Sept-batch2025/Python/tuple/task5.py
list = []
n = int(input("Enter element = "))
for i in range(0,n):
    num = int(input("Enter number = ")) 
    list.append(num)
t=tuple(list)

x = int(input("Enter Search Number = "))
print(t)
print(t.count(x))